apps/blog/src/app/global.css

```
@@ -2163,3 +2163,865 @@ g[data-top="true"] .vector-demo-point {
+  width: 100%;
+  height: 100%;
+  overflow: visible;
+}
+
+.sf-path-head {
+  position: absolute;
+  right: -1px;
+  top: 50%;
+  width: 0.375rem;
+  height: 0.375rem;
+  border-right: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)